### ISSUE TITLE
Provide some kind of contact tree

### ISSUE THREAD

**Issue by danrabbit** [elementary/website#968]
This is a common request from folks. We should provide some kind of contact tree that sorts folks out between appropriate channels. Basically we should offer an obvious "contact" link and then ask users what they're looking for and redirect them to the right thing:
- [ ] Help with elementary OS: StackExchange
- [ ] Help with my payment: mailto: payment at elementary.io
- [ ] I have a suggestion for this website: this issue tracker
- [ ] I have a suggestion for elementary OS: bugs.launchpad.net/elementary or maybe the help article on filing issues?
- [ ] I want to get involved with development: get involved page

If you check out the [ChicagoFire website](http://chicagofire.com/) (which is now maintained by someone else, so please I'm embarrassed I'm even linking this. It's not my fault that it's slow as hell now), I did something similar. Scroll to the footer and select "Contact". It shows a dialog with options.

**Comment by cassidyjames** [comment]
@danrabbit would adding a "Contact" item to the nav with a popover with those options make sense?

**Comment by danrabbit** [comment]
@cassidyjames I almost wonder if the footer would be better than the nav so people don't choose it as the very first thing. But yeah, I think a popover could work

**Comment by cianwalsh** [comment]
I can take a look at this if no one else is working on it.

**Comment by cassidyjames** [comment]
I agree that footer would be better. It's where we've been putting less-important but still need-to-be-accessible links. I wonder if this will require a slight redesign of the footer, though, as we've recently added two more links as well.

**Comment by gsumpster** [comment]
Shouldn't something like this be on the support page? 80% of the options listed above don't actually contact anyone.

I made a quick mockup and attached it here. I made a few assumptions here, didn't have any data to go off of, would be interested in some thoughts from the UX guys;

- the majority of clicks within the support page are going to the Installation guide
- the program specific guides either aren't being clicked or aren't providing users with what they were expecting (hard to collect data on, would be interested if it had data).

![artboard 1](https://cloud.githubusercontent.com/assets/3413011/17454919/714fc070-5b5c-11e6-82c4-d139ef938939.png)

Here's a link to the illustrator file if anyone would like to modify without recreating everything: http://adobe.ly/2aPrqVz

Would love some thoughts on this contact tree being on the support page and the mockup as a whole, hope this is useful. If this is OK, I'd be more than willing to implement this for the support page.
